scss/utils/helpers/_forms.scss

```diff
@@ -0,0 +1,26 @@
+////
+/// @group forms
+////
+
+/// resets form field styles
+/// @access public
+///
+/// @example scss
+/// 	input.reset {
+/// 		@include resetFieldStyles();
+/// 	}
+@mixin resetFieldStyles {
+	background-color: transparent;
+	border-width: 0;
+	height: auto;
+	margin: 0;
+	padding: 0;
+
+	&::-webkit-inner-spin-button,
+	&::-webkit-outer-spin-button {
+		-webkit-appearance: none;
+		-moz-appearance: none;
```

Comment:
`-moz-appearance` and `appearance` should probably be moved outside of the `&::-webkit-inner-spin-button` block